pod/perlop.pod

```diff
@@ -1517,11 +1519,22 @@ takes higher precedence.
 
 Then again, you could always use parentheses.
 
-Binary C<"xor"> returns the exclusive-OR of the two surrounding expressions.
-It cannot short-circuit (of course).
-
 There is no low precedence operator for defined-OR.
 
+=head2 Logical Exclusive Or
+X<operator, logical, xor> X<operator, logical, exclusive or> X<xor>
+
+Binary C<"xor"> returns the logical disjunction of the two surrounding
+expressions.  That means it returns 1 if either, but not both, are true.
```

Comment:
Does it return `1` or `true` (the boolean)?

Reply:
`true`.  Fixed